addons/resource/tests/__init__.py

```diff
@@ -1,4 +1,4 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 # -*- coding: utf-8 -*-
 
-from . import test_utils
+from . import test_resource_calendar, test_utils
```

Comment:
It's usually written like this, let's keep it consistent :smile: 
```suggestion
from . import test_resource_calendar
from . import test_utils
```